docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -0,0 +1,310 @@
+```
+
+```bash
+kubectl -n kube-system get pods
+
+NAME                                                    READY     STATUS    RESTARTS   AGE
+dns-controller-3497129722-rt4nv                         1/1       Running   0          7m
+etcd-server-events-ip-172-20-125-90.ec2.internal        1/1       Running   0          7m
+etcd-server-events-ip-172-20-48-12.ec2.internal         1/1       Running   0          3m
+etcd-server-events-ip-172-20-79-203.ec2.internal        1/1       Running   0          7m
+etcd-server-ip-172-20-125-90.ec2.internal               1/1       Running   0          7m
+etcd-server-ip-172-20-48-12.ec2.internal                1/1       Running   0          3m
+etcd-server-ip-172-20-79-203.ec2.internal               1/1       Running   0          7m
+kube-apiserver-ip-172-20-125-90.ec2.internal            1/1       Running   0          7m
+kube-apiserver-ip-172-20-48-12.ec2.internal             1/1       Running   0          3m
+kube-apiserver-ip-172-20-79-203.ec2.internal            1/1       Running   0          7m
+kube-controller-manager-ip-172-20-125-90.ec2.internal   1/1       Running   0          7m
+kube-controller-manager-ip-172-20-48-12.ec2.internal    1/1       Running   0          3m
+kube-controller-manager-ip-172-20-79-203.ec2.internal   1/1       Running   0          7m
+kube-dns-479524115-28zqc                                3/3       Running   0          8m
+kube-dns-479524115-7xv6b                                3/3       Running   0          6m
+kube-dns-autoscaler-1818915203-zf0gd                    1/1       Running   0          8m
+kube-proxy-ip-172-20-125-216.ec2.internal               1/1       Running   0          6m
+kube-proxy-ip-172-20-125-90.ec2.internal                1/1       Running   0          7m
+kube-proxy-ip-172-20-48-12.ec2.internal                 1/1       Running   0          3m
+kube-proxy-ip-172-20-79-203.ec2.internal                1/1       Running   0          7m
+kube-proxy-ip-172-20-92-185.ec2.internal                1/1       Running   0          7m
+kube-scheduler-ip-172-20-125-90.ec2.internal            1/1       Running   0          7m
+kube-scheduler-ip-172-20-48-12.ec2.internal             1/1       Running   0          3m
+kube-scheduler-ip-172-20-79-203.ec2.internal            1/1       Running   0          8m
+
+```
+
+
+## LAUNCH A SIMPLE REPLICATED APP ON THE CLUSTER.
+
+Before doing the tasks ahead, we created a simple "webservers" security group inside our KOPS's cluster VPC (using the AWS WEB-UI) allowing inbound port 80 and applied it to our two nodes (not the masters). Then, with the following command we proceed to create a simple replicated app in our coreos-based kops-launched cluster:
```

Comment:
Would be great to add a comment that this is not the recommended way to do things - that we should be using a service of Type=LoadBalancer or an Ingress.  But that it is useful to understand how things work :-)